llvm/test/CodeGen/AArch64/sms-acceptable-loop3.mir

```diff
@@ -1,80 +0,0 @@
-# RUN: llc --verify-machineinstrs -mtriple=aarch64 -o - %s -run-pass pipeliner -aarch64-enable-pipeliner -pipeliner-enable-copytophi=0 -debug-only=pipeliner 2>&1 | FileCheck %s
```

Comment:
I omitted CBZ because of the additional processing required to support it and because we did not see it used in typical loops to pipeline. If that support is found to be useful I would like to modify it.

The tests were changed to remain and marked as unsupported.
https://github.com/llvm/llvm-project/blob/004574d8fbe851ed10f651ff7e99919f40b0ff95/llvm/test/CodeGen/AArch64/sms-acceptable-loop4.mir#L4-L5

Reply:
Sounds good.

I would say it might be best to update the CHECK below instead or marking it unsupported. We can then check we don't do anything bad with the CBZ. The comment is good to keep in explaining that the instruction isn't supported.

Reply:
Thank you. Fixed to make sure it is not pipelined.